scripts/train/train.py

```diff
@@ -244,11 +245,24 @@ def main(cfg: DictConfig):
                                                           'eval_loader',
                                                           must_exist=False,
                                                           default_value=None)
-    icl_tasks_config: Optional[ListConfig] = pop_config(cfg,
+    icl_tasks_config: Optional[Union[ListConfig,
+                                     str]] = pop_config(cfg,
                                                         'icl_tasks',
                                                         must_exist=False,
                                                         default_value=None)
-
+    eval_gauntlet_config: Optional[Union[DictConfig,
```

Comment:
In order to not break existing yamls, please check for both `model_gauntlet` and `eval_gauntlet`, and emit a deprecation warning if `model_gauntlet` is used.